### DISCUSSION TITLE
how to use routr/routr-connect

### DISCUSSION THREAD

**Discussion by 406112767** [fonoster/routr#187] [Q&A] [answered]
I already installed routr/routr-connect, but I don‘t know how to use it. I can't find any document about it.
I want to add a gateway through routr/routr-connect.

**Comment by psanders** [comment]
@406112767, I'm sorry you are experiencing these issues. I'm currently working on preparing documentation for version [2.x.x](https://routr.io/docs/next/welcome) and aim to have it ready no later than Friday of next week. If you need this urgently, please consider obtaining the Official Handbook here: https://fonoster.gumroad.com/l/the-future-of-programmable-sip-servers.

**Reply by 406112767** [reply]
I bought the Official Handbook. but I don't know how to use rctl to connect apiserver. I can't login rctl.

**Reply by psanders** [reply]
I can help.

Do you mind sharing some details?

For example, are you still using K8s?

Are you seeing any errors?

**Reply by 406112767** [reply]
I performed the following steps：
1、kubectl -n sipnet port-forward svc/sipnet-routr-apiserver 51908
2、rctl login http://127.0.0.1:51908/api/v2beta1 -u admin -p changeit
the second command response:
Unable to perform request. Ensure server is up!

**Reply by psanders** [reply, marked as answer]
It seems you are using the old rctl. Would you instead install from npmjs.com/package/@routr/ctl

Installation steps are in the Getting Started session.

I'm sorry for the confusion.

**Reply by 406112767** [reply]
it works, thanks.
